src/session/Key.cc

```diff
@@ -4,9 +4,10 @@
 
 namespace zeek::session::detail {
 
-Key::Key(const void* session, size_t size, bool copy) : size(size)
+Key::Key(const void* session, size_t size, bool copy) : size(size), copied(copy)
```

Comment:
I also didn't understand Coverity 1452759 (the control flow it indicates doesn't seem possible), but this change also seems like undesired logic and causes reproducible CI failure in fuzzing tests -- the `copied` member seems to mean "already copied", so setting it before `CopyData()` is actually called prevents that from doing the actual copy.

Reply:
Fixed.